pkg/gcrane/copy.go

```diff
@@ -33,6 +37,27 @@ import (
 	"golang.org/x/sync/errgroup"
 )
 
+// These numbers are based on GCR's posted quotas:
+// https://cloud.google.com/container-registry/quotas
+// -  30k requests per 10 minutes.
+// - 500k requests per 24 hours.
+//
+// On error, we will wait for:
+// - 6 seconds (in case of very short term 429s from GCS), then
+// - 1 minute (in case of temporary network issues), then
+// - 10 minutes (to get around GCR 10 minute quotas), then fail.
```

Comment:
What's the `Cap: 1 * time.Hour` for?

Reply:
Basically to make sure I don't have an off-by-one error in my understanding of ExponentialBackoff. We don't really want to sit around for any longer than an hour, ever.